src/commands/scanner/rule/describe.ts

```diff
@@ -78,6 +79,7 @@ export default class Describe extends ScannerCommand {
 		return rules.map(r => {
 			const styledRule: DescribeStyledRule = {
 				...r,
+				runWith: r.isDfa ? 'scanner:run:dfa' : 'scanner:run',
```

Comment:
Is there a way to get the command string directly from the command typescript class? This looks good as well - just checking if we can avoid maintaining this in the future.

Reply:
Maybe. I'll check.